android-components/components/feature/search/src/main/assets/searchplugins/daum-kr.xml

```diff
@@ -3,7 +3,7 @@
    - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
 
 <SearchPlugin xmlns="http://www.mozilla.org/2006/browser/search/">
-<ShortName>다음지도</ShortName>
+<ShortName>다음</ShortName>
```

Comment:
The name changed here to reflect what the search engine is not actually searching a map (to reflect what Desktop does).